pkg/process/runner/submitter.go

```diff
@@ -122,30 +123,30 @@ func NewSubmitter(hostname string) (*CheckSubmitter, error) {
 	if err != nil {
 		return nil, err
 	}
-	processForwarderOpts := forwarder.NewOptionsWithResolvers(resolver.NewSingleDomainResolvers(apicfg.KeysPerDomains(processAPIEndpoints)))
+	processForwarderOpts := forwarder.NewOptionsWithResolvers(config.Datadog, resolver.NewSingleDomainResolvers(apicfg.KeysPerDomains(processAPIEndpoints)))
```

Comment:
We're tying to avoid using the global `config.Datadog` if at all possible. Do you mind using the injected one from the runner's `dependencies` struct. I currently have https://github.com/DataDog/datadog-agent/pull/15928 that i'm probably going to merge either today or tomorrow which removes all of the usages.

Reply:
Fixed by https://github.com/DataDog/datadog-agent/pull/16563/commits/d2978cb17e15ebfd84513ea3841580fa3097e84e.